testdata/scripts/extract.txt

```diff
@@ -616,6 +651,11 @@ Permanently delete an account.
 
 Retrieves all data from an account, selected by the account_id you supplied.
 
+```sh
+curl -X GET \
+    https://openbank.com/v1/accounts/{AccountID} \
+    -H 'Authorization: Bearer USE_YOUR_TOKEN' \
```

Comment:
This is what I meant in the other comment - the trailing `\` will confuse some users.